controller/src/beerocks/master/db/node.cpp

```diff
@@ -297,3 +297,89 @@ bool node::set_type(beerocks::eType type_)
     }
     return false;
 }
+
+bool node::link_metrics_data::add_transmitter_link_metric(
+    std::shared_ptr<ieee1905_1::tlvTransmitterLinkMetric> TxLinkMetricData)
```

Comment:
The coding style says that we use lower_snake_case, so that should be `tx_link_metric_data`.

Some of the new code still uses CamelCase, but that's just to be consistent with the surrounding code.

Same for other new variables introduced in this PR.

I should have made this comment earlier, of course.

(nitpick)